### ISSUE TITLE
Week #8 goals

### ISSUE THREAD

**Issue by gythialy** [qlcchain/go-qlc#129]
- VM @gythialy 
   - #127 write and review Smart Contract authentication abstract design doc
- CLI  @zengchen221 
     - #126 verify feasibility of ishell library for creating interactive cli
- P2P @pwc5232759 
    - #122 
    - #123 
    - #124 
    - #125
- Consensus @ariesunny 
   - #128 Summary a doc about DPOS consensus
- RPC @zengchen221 
   - #113 review

**Comment by gythialy** [comment]
#123 #124 blocked by #135, postponed to next week.
